Security in WordPress/Security.md

```diff
@@ -187,5 +187,42 @@ _______________________________________________________________
 [+] Requests made: 61
 [+] Memory used: 35.848 MB
 ```
+## Code snippets
 
+### Remove WP version from generator in head
+```
+  /**
+   * Return empty string in the 'wp_generator'
+   *
+   * @return string Returns an empty string.
+   */
+  public function empty_generator_version() : string {
+    return '';
+  }
+```
+Hooks into:
+```
+$this->loader->add_filter( 'the_generator', $security, 'empty_generator_version' );
```

Comment:
We should maybe just add what `$security` is (a class where the method is located).

Reply:
Added 👍